com.discordapp.Discord.json

```diff
@@ -23,7 +23,8 @@
         "--env=XDG_CURRENT_DESKTOP=Unity",
         "--talk-name=org.freedesktop.portal.Fcitx",
         "--talk-name=org.kde.StatusNotifierWatcher",
-        "--talk-name=com.canonical.AppMenu.Registrar"
+        "--talk-name=com.canonical.AppMenu.Registrar",
+        "--talk-name=com.canonical.indicator.application"
```

Comment:
This permission probably does nothing. Unity doesn't even exist any more.

Reply:
Yeah, thought as much.